packages/render-fidelity-tools/package.json

```diff
@@ -43,6 +43,7 @@
     "mkdirp": "^1.0.4",
     "pngjs": "^6.0.0",
     "puppeteer": "^10.1.0",
+    "rhodonite": "github:actnwit/RhodoniteTS#d42809b7b20976d35c9ef682d749b9c2b2887fb8",
```

Comment:
Is there a chance you'd be willing to publish an npm release (even an alpha or something) rather than using a specific commit? It's nice ideally to be able to look at release notes and see what changes are expected when upgrading.

Reply:
Sure. I'll do it.

Reply:
@elalish I'm currently having an unknown problem with rhodonite that is unable to publish the new version to npm. I am sorry, but can I keep this SHA designation for the time being until the problem is resolved?

Reply:
Sure, that's fine for now. I'll just wait on the images.

Reply:
@elalish I pushed the rendering images.

Reply:
Have you compared the images to the other renderers? It looks like the camera / lighting / tone mapping is not consistent yet. I understand shader differences, but at least the scene setup should match before we merge this. I would also exclude any scenarios where your renderer is deeply disfunctional and add them back in as you release improvements.

Reply:
@elalish Sure. I will check the other renderer's results and make some more adjustments.

Reply:
@elalish Adjusted lighting environment and fixed bugs. I think I have minimized the differences from output images of other libraries. Also excluded scenarios that did not work enough.
I succeeded in publishing 0.8.5 to npm, so I replaced rhodonite with it.